pkgs/by-name/fl/flashgbx/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, copyDesktopItems
+, makeDesktopItem
+, qt6
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "flashgbx";
+  version = "3.36";
+
+  src = fetchFromGitHub {
+    repo = "FlashGBX";
+    owner = "lesserkuma";
+    rev = "${version}";
```

```suggestion
    rev = version;
```

there's no need to interpolate here

---

Another good catch, thanks.